libnavui-dropin/src/main/res/layout/mapbox_info_panel_layout.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="vertical"
+    android:background="@drawable/mapbox_bg_info_panel"
+    android:elevation="8dp">
+
+    <FrameLayout
+        android:id="@+id/infoPanelHeader"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        tools:minHeight="@dimen/mapbox_infoPanel_peekHeight"
+        tools:background="#00ccff" />
+
+    <FrameLayout
+        android:id="@+id/infoPanelContent"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:layout_constraintTop_toBottomOf="@id/infoPanelHeader"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        tools:background="#ccffcc"
+        tools:minHeight="200dp" />
+
+    <View
+        style="@style/DropInStyleDragHandle"
+        android:id="@+id/handle"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent" />
```

Comment:
@abhishek1508 Just FYI. I moved this drag handle view from the header view.
This will allow developers to:
- replace or hide it in their custom info panel layout while using the default header
- change the position of the `infoPanelHeader` layout (eg. when placing an additional view above header view)

Reply:
Nice 👍🏻